packages/plugins/src/foundry/utils/index.ts

```diff
@@ -89,40 +123,116 @@ export const makeGetConfigArtifacts = (
         time: fs.statSync(`${buildInfoPath}/${fileName}`).mtime.getTime(),
```

Comment:
> I think we should use `path.join` here instead of using a platform-specific delimiter, `/`

Ik you didn't touch this line in this PR, but we should probably fix this too